editor/src/core/workers/parser-printer/uid-fix.ts

```diff
@@ -595,21 +617,31 @@ export function fixJSXElementUIDs(
   )
 
   // Carry the UID of the prop that maybe set over as well.
-  let dataUIDPropUID: string | undefined = undefined
-  if (oldElement != null && isJSXElement(oldElement)) {
-    const oldDataUIDProp = getJSXAttribute(oldElement.props, 'data-uid')
-    const newDataUIDProp = getJSXAttribute(fixedProps, 'data-uid')
-    if (oldDataUIDProp != null && newDataUIDProp != null) {
-      if (fixUIDsState.mutableAllNewUIDs.has(oldDataUIDProp.uid)) {
-        dataUIDPropUID = newDataUIDProp.uid
-      } else {
-        dataUIDPropUID = updateUID(
-          identityOptic<string>(),
-          oldDataUIDProp.uid,
-          { ...fixUIDsState, uidUpdateMethod: 'forced-update' },
-          newDataUIDProp.uid,
-        )
-      }
+  let dataUIDPropUID: string
+  if (newDataUIDProp == null) {
+    // Backup case for where there is no `data-uid` prop.
+    dataUIDPropUID = generateConsistentUID(
```

Comment:
Shouldn't `updateUID` be called here? Is the mapping updated in this branch?

Reply:
So it can't be there because we have no uids to update with, but it should add the UID to `mutableAllNewUIDs`, fixed in 67a36bd03.